packages/new-polymath-issuer/src/pages/Home.tsx

```diff
@@ -1,3 +1,28 @@
-import React from 'react';
+import { polyClient } from '~/lib/polymath';
+import React, { Component } from 'react';
+import { browserUtils } from '@polymathnetwork/sdk';
 
-export const HomePage = () => <div>[Home Page]</div>;
+class Container extends Component {
+  public async componentDidMount() {
+    await browserUtils.getCurrentAddress();
```

Comment:
This is just demo code, the reason to do this here is because I didn't want to mess with access control for a code example. `getCurrentAddress()` will trigger the `.enable` mechanism on Metamask if we are in the browser.